home-work-app/src/app/components/modals/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,30 @@
+import { EditForm, EditFormFieldset, EditFormLegend, Label, Input } from './EditFormStyleSet.styled';
+import { ButtonPanel, Button } from '../CommonModalTemplate/ButtonPanel.styled';
+import { stylesConfig } from '../../../configs/style.configs';
+
+type MovieFormProps = {
+    legend: string,
+    fields: string[]
+};
+
+export const MovieForm: React.FC<MovieFormProps> = (props: MovieFormProps) => (
```

Comment:
Maybe `MovieModalForm`?

Reply:
I think not. I've left an option to use it outside the modal window